experimental/trusted_shuffler/scripts/run

```diff
@@ -2,12 +2,16 @@
 
 readonly SCRIPTS_DIR="$(dirname "$0")"
 
+readonly K=2
+readonly QPS=10
+readonly ROUNDS=2
+
```

Comment:
If only we could use a real programming language to coordinate the execution of various things... :upside_down_face:

Reply:
`xtask` won't be able to spawn it on multiple machines (which is required to reach large QPS numbers for testing)

Reply:
How does this script spawn on multiple machines? Or will it do it in the future?